cmd/aro/rp.go

```diff
@@ -45,7 +45,6 @@ func rp(ctx context.Context, log, audit *logrus.Entry) error {
 	} else {
 		keys = []string{
 			"ACR_RESOURCE_ID",
-			"ADMIN_API_CLIENT_CERT_COMMON_NAME",
```

Comment:
could consider entirely dropping this commit given that we are setting up the ARM and admin authorizers against a developer CA